src/module/module.py

```diff
@@ -23,10 +26,14 @@ def module_main(received_data: any) -> str:
 
     """
 
-    log.debug("Outputting ...")
+    log.debug("Writing to Airtable ...")
 
     try:
-        # YOUR CODE HERE
+        if type(received_data) is list:
+            for data in received_data:
+                table.create(data)
+        else:
+            table.create(received_data)
```

Comment:
consider the following scenario: you want to add 5 elements to the table and the 3rd (of 5) labels is erroneous. in the current implementation this would mean that you are going to add the first two elements and exit on the third. I think in this case it would be better either to just skip the 3rd element and continue with the 4th and 5th OR revert adding the first two once the error is encountered.

Reply:
Yes, you are right. I tried some implementations with response from the server but still didn't work well. In this case I implemented `table.batch_create` so it will either successfully write all data or reject all data. Seems like the best thing to do. Also they have this `typecast` flag but it only casts data from string to sth else and does not cast from sth to sth.